apps/console/src/features/identity-providers/components/wizards/organization-enterprise/organization-enterprise-authentication-provider-create-wizard-content.tsx

```diff
@@ -0,0 +1,255 @@
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+    OrganizationEnterpriseAuthenticationProviderCreateWizardFormErrorValidationsInterface,
+    OrganizationEnterpriseAuthenticationProviderCreateWizardFormValuesInterface
+} from "./organization-enterprise-authentication-provider-create-wizard";
+import { getIdentityProviderList } from "../../../api";
+import { IdentityProviderManagementConstants } from "../../../constants";
+import { IdentityProviderListResponseInterface, IdentityProviderTemplateInterface } from "../../../models";
+import { handleGetIDPListCallError } from "../../utils";
+
+/**
+ * Proptypes for the GitHub Authentication Provider create wizard content.
+ */
+interface OrganizationEnterpriseAuthenticationProviderCreateWizardContentPropsInterface
+    extends IdentifiableComponentInterface {
+    /**
+     * Trigger form submit.
+     * @param {() => void} submitFunctionCb - Callback.
+     */
+    triggerSubmission: (submitFunctionCb: () => void) => void;
+    /**
+     * Trigger previous page.
+     * @param {() => void} previousFunctionCb - Callback.
+     */
+    triggerPrevious: (previousFunctionCb: () => void) => void;
+    /**
+     * Callback to change the wizard page,
+     * @param {number} pageNo - Page Number.
+     */
+    changePageNumber: (pageNo: number) => void;
+    /**
+     * IDP template.
+     */
+    template: IdentityProviderTemplateInterface;
+    /**
+     * Total wizard page count.
+     * @param {number} pageCount - Page number.
+     */
+    setTotalPage: (pageCount: number) => void;
+    /**
+     * Callback to be triggered for form submit.
+     * @param values
+     */
+    onSubmit: (values: OrganizationEnterpriseAuthenticationProviderCreateWizardFormErrorValidationsInterface) => void;
+}
+
+/**
+ * Authentication Provider Create Wizard content component.
+ *
+ * @param {OrganizationEnterpriseAuthenticationProviderCreateWizardContentPropsInterface} props - Props injected to the component.
+ *
+ * @return {React.ReactElement}
+ */
+export const OrganizationEnterpriseAuthenticationProviderCreateWizardContent:
+    FunctionComponent<OrganizationEnterpriseAuthenticationProviderCreateWizardContentPropsInterface> = (
+        props: OrganizationEnterpriseAuthenticationProviderCreateWizardContentPropsInterface
+    ): ReactElement => {
+
+        const {
+            triggerSubmission,
+            triggerPrevious,
+            changePageNumber,
+            template,
+            setTotalPage,
+            onSubmit,
+            ["data-componentid"]: componentId
+        } = props;
+
+        const { t } = useTranslation();
+
+        const [ idpList, setIdPList ] = useState<IdentityProviderListResponseInterface>({});
+        const [ isIdPListRequestLoading, setIdPListRequestLoading ] = useState<boolean>(undefined);
+
+        /**
+     * Loads the identity provider authenticators on initial component load.
```

Comment:
Do we need a doc comment for this? Suggestion to use a single line comment instead.

Reply:
Used the same conventions used for other functions and useEffect hooks